providers/porkbun/porkbunProvider.go

```diff
@@ -43,14 +48,31 @@ func newDsp(conf map[string]string, metadata json.RawMessage) (providers.DNSServ
 
 // newPorkbun creates the provider.
 func newPorkbun(m map[string]string, _ json.RawMessage) (*porkbunProvider, error) {
-	c := &porkbunProvider{}
+	c := &porkbunProvider{
+		maxAttempts: defaultMaxAttempts,
+	}
 
 	c.apiKey, c.secretKey = m["api_key"], m["secret_key"]
 
 	if c.apiKey == "" || c.secretKey == "" {
 		return nil, errors.New("missing porkbun api_key or secret_key")
 	}
 
+	if maxAttempts, ok := m["max_attempts"]; ok && maxAttempts != "" {
+		i, err := strconv.Atoi(maxAttempts)
+		if err != nil {
+			return nil, errors.New("porkbun: invalid max_attempts: must be a whole number")
```

Comment:
Please output the invalid value too.  It makes it easier for the user to find what the change in `creds.json`.

Reply:
Updated. It now outputs

```
porkbun: invalid max_attempts "a": must be a whole number
```